data/1.0/_index.md

```diff
@@ -34,24 +34,41 @@ garage.save(ferrari);
 
 ### New features, enhancements or additions
 
-* The CrudRepository feature
-* Allows pagination on repository with PageableRepository
+* Define the Repository pattern
+* Allows pagination on Repository
+* Enhance type safety with a StaticMetamodel
+* Define platform integrations with CDI, Persistence, NoSQL, Transactions, and Validation
+* Produce a TCK with the ability to run against the following matrix:
+  * Profiles: none, core, web, full
```

Comment:
This is very confusion and they don't match with the formal term. I would rather delete them.